### PR TITLE
Improve labels and help text for cloud and pod templates cap

### PR DESCRIPTION
**Pull request by mbarbero**
The current description is mentioning container while it's really pod.

### CODE REVIEW

File: src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java [hunk 1]
```diff
@@ -521,19 +528,11 @@ private boolean addProvisionedSlave(@Nonnull PodTemplate template, @CheckForNull
...
-            LOGGER.log(Level.INFO,
-                    "Total container cap of {0} reached, not provisioning: {1} running or pending in namespace {2} with Kubernetes labels {3}",
-                    new Object[] { containerCap, allActiveSlavePods.size() + scheduledCount, templateNamespace, getLabels() });
-            return false;
-        }
-
         if (activeTemplateSlavePods != null && allActiveSlavePods != null && template.getInstanceCap() <= activeTemplateSlavePods.size() + scheduledCount) {
             LOGGER.log(Level.INFO,
-                    "Template instance cap of {0} reached for template {1}, not provisioning: {2} running or pending in namespace {3} with label \"{4}\" and Kubernetes labels {5}",
+                    "Maximum number of concurrently running agent pods ({0}) reached for template '{1}' in Kubernetes Cloud '{6}', not provisioning: {2} running or pending in namespace {3} with label \"{4}\" and Kubernetes labels {5}",
```
Comment (carlossg) [inline src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java]: talking from memory the single quotes will prevent the expansion of `{1}` and `{6}`

File: src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java [hunk 2]
```diff
@@ -511,6 +511,13 @@ private boolean addProvisionedSlave(@Nonnull PodTemplate template, @CheckForNull
                     .collect(Collectors.toList());
         }
 
+        if (allActiveSlavePods != null && containerCap <= allActiveSlavePods.size() + scheduledCount) {
+            LOGGER.log(Level.INFO,
+                    "Maximum number of concurrently running agent pods ({0}) reached for Kubernetes Cloud '{4}', not provisioning: {1} running or pending in namespace '{2}' with Kubernetes labels '{3}'",
```
Comment (carlossg) [inline src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java]: same comment about single quotes

File: src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java (hunk 1 above)
Comment (mbarbero) [reply src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java]: Done.

File: src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java (hunk 2 above)
Comment (mbarbero) [reply src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java]: Done.

**Pull request merged**